.github/workflows/build-test.yml

```diff
@@ -105,4 +102,40 @@ jobs:
           flags: unittests # optional
           name: codecov-umbrella # optional
           fail_ci_if_error: true # optional (default = false)
-          verbose: true # optional (default = false)
\ No newline at end of file
+          verbose: true # optional (default = false)
+
+  mnist-test: 
+    name: Test MNIST Notebook 
+    runs-on: [self-hosted, Linux, ubuntu-22-04]
+    needs: unit-test
+
+    if: github.event.pull_request.draft == false
+    steps:
+      - name: Checkout to commit
+        uses: actions/checkout@v3
+
+      - name: Filter module related changes
+        uses: dorny/paths-filter@master
+        id: filter
+        with:
+          filters: |
+            module:
+              - 'fedbiomed/**'
+              - 'scripts/**'
+              - 'envs/development/conda/**'
+              - 'envs/development/docker/**'
+              - 'notebooks/**'
+              - '.github/**'
+
+      - name: Run MNIST Test - Master
+        if: steps.filter.outputs.module == 'true'
+        run: |
+          source ~/.bashrc
+          ./scripts/fedbiomed_run network
```

Comment:
As we suppressed the use of `./scripts/CI_build` that was doing a `./scripts/configure_conda` I suspect we need at least a : `./scripts/configure_conda node` (network and researcher were done by unit tests ...).

If not, the conda envs are not properly updated.

Reply:
Hi @mvesin, 
Thanks for reminding. I am going to add it.